### DISCUSSION TITLE
Connection reset by peer

### DISCUSSION THREAD

**Discussion by SoyDiego** [rustdesk/rustdesk#9367] [Q&A]
Hi, I installed RustDesk and I can see the login screen of my remote desktop.
After login, I can see the desktop but 4 seconds later show me "Connection reset by peer" and black screen.
Always I need to go to the top toolbar and restart the remote desktop.

I tried a lot of options in Settings and tried changing the codecs but I can't use it.

Any help?

Thanks!

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#connection-reset-by-the-peer
